components/table/src/table.tsx

```diff
@@ -9,9 +9,7 @@ const Table = forwardRef<HTMLTableElement, HTMLAttributes<HTMLTableElement>>(({
 Table.displayName = "Table";
 
 const TableHeader = forwardRef<HTMLTableSectionElement, HTMLAttributes<HTMLTableSectionElement>>(
-	({ className, ...props }, ref) => (
-		<thead ref={ref} className={cx("bg-background [&_tr]:border-b", className)} {...props} />
-	),
+	({ className, ...props }, ref) => <thead ref={ref} className={cx("bg-base [&_tr]:border-b", className)} {...props} />,
```

Comment:
I think we'll wanna add these tokens to the table as well :v:

Reply:
Yeah probably. I didn't actually touch the table yet. Oh, now seeing that it uses `bg-background`. Yeah I think this will probably be `card`